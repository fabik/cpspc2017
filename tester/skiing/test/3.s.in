1
169838237 783158187 -907790640 -700176197
-340179415 844091241 81129190
