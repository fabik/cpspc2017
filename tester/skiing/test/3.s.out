3
169838237 783158187
-340179415 81129190
-907790640 -700176197
